modules/prebidServerBidAdapter/index.js

```diff
@@ -902,6 +902,20 @@ Object.assign(ORTB2.prototype, {
     addBidderFirstPartyDataToRequest(request, s2sBidRequest.ortb2Fragments?.bidder || {});
 
     request.imp.forEach((imp) => this.impRequested[imp.id] = imp);
+
+    if (typeof request.ext.prebid.floors === 'object' && request.ext.prebid.floors.enabled) {
+      const bidfloorArray = [];
+      request.imp.forEach(imp => {
+        if (imp.bidfloor) {
+          imp.ext.prebid.floors = {
+            floorMin: imp.bidfloor
+          }
+          bidfloorArray.push(imp.bidfloor);
+        }
+      });
+      request.ext.prebid.floors.floorMin = Math.min(...bidfloorArray);
```

Comment:
this loops over floors twice more, which is rather inefficient compared to  the zero that are required. You could keep track of the minimum as we calculate the floors, and here just append it to the request if `ext.prebid.floors.enabled` is true.

Reply:
thanks @dgirardi.  would the changes for my latest commit work?  also, one additional question, for each imp would i still need to assign `imp.ext.prebid.floors.floorMin = imp.bidfloor`?  (as seen in the comment above)